### PR TITLE
INFCT-76 - Update the documentation for the knife-vrealize plugin

### PR DESCRIPTION
**Pull request by ashiqueps**
Signed-off-by: Ashique Saidalavi <ashique.saidalavi@progress.com>

## Description
The workflow of the knife-vrealize plugin has been changed since we added support for the vRA8.x in this pr https://github.com/chef/knife-vrealize/pull/62. This PR will update the documentation to reflect the new changes in the knife-vrealize plugin. 

[Please describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/vmware.md [hunk 1]
```diff
@@ -157,7 +157,8 @@ WARNING: Deleted client example-01
 [\[GitHub\]](https://github.com/chef-partners/knife-vrealize)
 
 - Supports both vRealize Automation and vRealize Orchestrator
-- Supports vRealize Automation 7.0+
+- Supports vRealize Automation 8.x. Version 7.0.0 and later will support vRA 8.x.
```
Comment (IanMadd) [inline content/vmware.md:160]: Are you about to release knife-vrealize 7?

**Comment by ashiqueps** [comment]
Hi @IanMadd,

Could you please review this PR? The version `7.0.0` is now available in the rubygems.

**Comment by IanMadd** [comment]
@ashiqueps can you allow me to make changes to the fork/branch. There's some changes I need to make.

**Comment by ashiqueps** [comment]
Sure @IanMadd, I've added you as a collaborator in my fork. Please let me know if you are having issues with accessing it.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**